Libraries/src/Amazon.Lambda.ConnectEvents/Amazon.Lambda.ConnectEvents.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\..\..\buildtools\common.props" />
+
+  <PropertyGroup>
+    <Description>Amazon Lambda .NET Core support - Amazon Connect package.</Description>
+    <TargetFrameworks>netstandard2.0;netcoreapp3.1</TargetFrameworks>
```

Comment:
You are not using any features of .NET Core 3.1 so you just need to target netstandard2.0. If you had to add some System.Text.Json attributes to customize the marshaling then you would need to add the netcore3.1 target framework.

Reply:
Fixed